### PR TITLE
Addinglegos updated automation

### PR DESCRIPTION
**Pull request by dougsillars**
i hgave updated the github action to add a list of all the Actions for each provider.  
Previously, the AWS directory readme had a list of all the xRunBooks.  Now it also has a list of all the Actions available for AWS (and this holds for all of the directories)

### CODE REVIEW

**Review by jayasimha-raghavan-unskript — COMMENTED** [review]
Please remove .ipynb_checkpoints folder. This is a temporary folder which gets generated when VSCode or any jupyter lab extension is loaded.

**Review by jayasimha-raghavan-unskript — CHANGES_REQUESTED** [review]
Please take care of minor comment.

File: .ipynb_checkpoints/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<img align="center" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding: 5px">
+
+ # .ipynb_checkpoints xRunBooks
```
[the remark this answers is not in the document]
Comment (dougsillars) [reply .ipynb_checkpoints/README.md]: added

**Review by jayasimha-raghavan-unskript — APPROVED** [review]
Looks good to me

**Pull request merged**